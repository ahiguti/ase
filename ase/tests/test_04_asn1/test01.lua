
local Test = ASE.CPP['ASETestCPP']
local arr = { "foo", "foobar", "baz" }
local s = Test.ToXML(arr)
print(s)

