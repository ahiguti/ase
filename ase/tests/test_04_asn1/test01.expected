<ASERemoteVariant><arrval><strval>foo</strval><strval>foobar</strval><strval>baz</strval></arrval></ASERemoteVariant>
